### PR TITLE
[Text Analytics] Make improvements to samples

### LINKED ISSUE #23105: [Text Analytics] Update samples text
The text used is too long and don't fit in the boxes in GitHub:
```
string document = @"Windows 365 was in the works before COVID-19 sent companies around the world on a scramble to secure solutions to support employees suddenly forced to work from home, but “what really put the firecracker behind it was the pandemic, it accelerated everything,” McKelvey said. She explained that customers were asking, “’How do we create an experience for people that makes them still feel connected to the company without the physical presence of being there?”
                    In this new world of Windows 365, remote workers flip the lid on their laptop, bootup the family workstation or clip a keyboard onto a tablet, launch a native app or modern web browser and login to their Windows 365 account.From there, their Cloud PC appears with their background, apps, settings and content just as they left it when they last were last there – in the office, at home or a coffee shop.
                    And then, when you’re done, you’re done.You won’t have any issues around security because you’re not saving anything on your device,” McKelvey said, noting that all the data is stored in the cloud.
                    The ability to login to a Cloud PC from anywhere on any device is part of Microsoft’s larger strategy around tailoring products such as Microsoft Teams and Microsoft 365 for the post-pandemic hybrid workforce of the future, she added. It enables employees accustomed to working from home to continue working from home; it enables companies to hire interns from halfway around the world; it allows startups to scale without requiring IT expertise.
                    “I think this will be interesting for those organizations who, for whatever reason, have shied away from virtualization.This is giving them an opportunity to try it in a way that their regular, everyday endpoint admin could manage,” McKelvey said.
                    The simplicity of Windows 365 won over Dean Wells, the corporate chief information officer for the Government of Nunavut. His team previously attempted to deploy a traditional virtual desktop infrastructure and found it inefficient and unsustainable given the limitations of low-bandwidth satellite internet and the constant need for IT staff to manage the network and infrastructure.
                    We didn’t run it for very long,” he said. “It didn’t turn out the way we had hoped.So, we actually had terminated the project and rolled back out to just regular PCs.”
                    He re-evaluated this decision after the Government of Nunavut was hit by a ransomware attack in November 2019 that took down everything from the phone system to the government’s servers. Microsoft helped rebuild the system, moving the government to Teams, SharePoint, OneDrive and Microsoft 365. Manchester’s team recruited the Government of Nunavut to pilot Windows 365. Wells was intrigued, especially by the ability to manage the elastic workforce securely and seamlessly.
                    “The impact that I believe we are finding, and the impact that we’re going to find going forward, is being able to access specialists from outside the territory and organizations outside the territory to come in and help us with our projects, being able to get people on staff with us to help us deliver the day-to-day expertise that we need to run the government,” he said.
                    “Being able to improve healthcare, being able to improve education, economic development is going to improve the quality of life in the communities.”";
```

### LINKED ISSUE #32735: [Text Analytics] Remove unnecessary manual polling from samples

### PR DESCRIPTION
**Pull request by joseharriaga**
Resolves: #32735 and #23105

These are mostly cosmetic changes to make our samples more consistent and easier to read:
* Removes verbatim strings from our samples, which were sending the wrong inputs.
* Replaces instances of unnecessary manual polling with automatic polling.
* Fixes linting warnings.
* Miscellaneous cleanup so all samples follow the same patterns and code style, making them easier understand as a whole.

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md [hunk 1]
```diff
@@ -11,41 +11,42 @@ description: Samples for the Azure.AI.TextAnalytics client library
...
+* [Abstractive Summarization](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample13_AbstractiveSummarize.md)
 
 ## Advanced samples
-- [Analyze Sentiment with Opinion Mining](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2.1_AnalyzeSentimentWithOpinionMining.md)
-- [Running multiple actions in one or more documents](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_AnalyzeActions.md)
-- [Recognize PII Entities with specific categories](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample5_RecognizePiiEntitiesWithCategoriesFilter.cs)
-- [Polling Long Running Operations](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_LROPolling.md)
-- [Mock client](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_MockClient.md)
 
-[README]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/README.md
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md:51]: README link removed in purpose?

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample11_DynamicClassify.md [hunk 2]
```diff
@@ -35,31 +37,30 @@ List<string> categories = new()
 
 try
 {
-    Response<ClassificationCategoryCollection> response = client.DynamicClassify(document, categories);
+    Response<ClassificationCategoryCollection> response = await client.DynamicClassifyAsync(document, categories);
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample11_DynamicClassify.md:40]: change from sync to async?

**Review by mssfang — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md (hunk 1 above)
Comment (joseharriaga) [reply sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md:51]: Yes, it was not used in this document, so I removed it. But on second thought, I can see how linking to the README could be useful. I'll add it back and write a sentence that uses the link. 😊

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample11_DynamicClassify.md (hunk 2 above)
Comment (joseharriaga) [reply sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample11_DynamicClassify.md:40]: Fixed!

**Pull request merged**